openapi/__tests__/definitions.test.ts

```diff
@@ -176,7 +175,7 @@ describe("NewMessage definition", () => {
     );
   });
 
-  it("should decode NewMessage with content and payment data but without payee", () => {
+  it("should NOT decode NewMessage with content and payment data but without payee", () => {
```

Comment:
So are you saying that every message containing a payment data should require a payee? even if the payee is the sender itself?

Reply:
Yep, in case payee is the sender itself we must force explicitly in payee field :)

Reply:
Ok, I remember we didn't want to make the sender define this field if no needed, because of backward compatibility.

Reply:
This field is optional for sender, but when we save message in our storage we must enforce if it is undefined :D

Reply:
Why did you say that the payee is optional for sender?  
I mean: `NewMessage` is the model needed by the [message creation API](https://github.com/pagopa/io-functions-services/blob/022914ce3bf322db929ea293cefcff4937052315/openapi/index.yaml#L100).
With this test I understand that NewMessage with content and payment data but without payee should fail the decoding part. So how can a service send a message with content and payment data with optional payee? 
I'm sure I'm missing something, I'm sorry

Reply:
> NewMessage is the model needed by the message creation API.
Nope `NewMessage` is the model used to store document in cosmosDB as well as a `CosmosDBModel`. The message Creation Api refers to a separated [definition](https://github.com/pagopa/io-functions-services/blob/022914ce3bf322db929ea293cefcff4937052315/openapi/index.yaml#L964) :)

Reply:
Actually the definition you linked is generated by the template. If you watch within the template you can see that NewMessage refers to io-function-commons one [here](https://github.com/pagopa/io-functions-services/blob/022914ce3bf322db929ea293cefcff4937052315/openapi/index.yaml.template#L647)

Reply:
Ok, now I got your point, I was misleading with [this](https://github.com/pagopa/io-functions-commons/blob/a6babf2322731a9cac975b4f2fb2668f9e407d16/src/models/message.ts#L128) type . So, no problem fixed in [287d057](https://github.com/pagopa/io-functions-commons/pull/158/commits/287d057e7825a647b9fcfb53e3781ef95e0d6833) :)